app/modules/transcribe/annotations/annotations.factory.js

```diff
@@ -57,8 +60,36 @@ function AnnotationsFactory(localStorageService) {
         return annotation;
     }
 
+
+    function checkVariants(annotation) {
+        //splits string by whitespace (different encodings)
+        var lemmas = annotation.text.split(/\s|&nbsp;/g);
+        var results = [];
```

Comment:
Why are you keeping `results` here? It's not available outside this function, and it's not added to the annotation or anything that I can see..?